{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "778d1dde_260d0f9a",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000009
      },
      "writtenOn": "2023-08-17T02:35:01Z",
      "side": 1,
      "message": "Okay, so the CI failure is due to a couple of failing test cases. I\u0027ll try to track those down when I have a moment.",
      "revId": "b9de5e7e689f976c5b8c5f525fbdc9a29623b440",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "3b50eef8_98ab5585",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000000
      },
      "writtenOn": "2023-08-17T02:50:49Z",
      "side": 1,
      "message": "I was planning on getting back to it and haven\u0027t yet.  I suspect I have the wrong mix of objects that go into one of the test cases and am missing some mocks.",
      "parentUuid": "778d1dde_260d0f9a",
      "revId": "b9de5e7e689f976c5b8c5f525fbdc9a29623b440",
      "serverId": "adbd0a64-1f21-4d83-b585-671fe73cb6e4"
    }
  ]
}